src/decirest_single_handler.erl

```diff
@@ -225,4 +224,10 @@ resource_exists_default(Req, State = #{module := Module}) ->
   {Res, ReqNew, StateNew} = decirest:call_mro(resource_exists, Req, State, true, Continue),
   {maps:get(Module, Res, false), ReqNew, StateNew}.
 
+is_exported(Module, Function, ArityList) when is_list(ArityList) ->
+  lists:any(fun(R) -> R end,
+    [is_exported(Module, Function, Arity) || Arity <- ArityList ]
+  );
 
+is_exported(Module, Function, Arity) ->
+  erlang:function_exported(Module, Function, Arity).
```

Comment:
why implement this one agin, if for shortcut can't we just do
```erlang
is_exported(Module, Function, Arity) ->
  decirest_handler_lib:is_exported(Module, Function, Arity).
```

Reply:
I missed to remove it when I moved it to the lib. Will fix